Automattic/jetpack
# Social: Remove unnecessary social connection feature checks

> Our feature checks are getting a bit unwieldy. Some like the Instagram connection have been launched for a year or more. We should remove the plugin version checks and other hacks so they're treated like any other feature. We can then remove the code from Jetpack, and eventually remove the feature entirely.

## Proposed changes:

* Remove the unnecessary feature checks for social connections

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to connections management
* Confirm that you see Instagram, Nextdoor and Mastodon services just like any other services.
* Open social previews in the editor
* Confirm that you see Instagram, Nextdoor and Mastodon services just like any other services.
